src/items/modules.md

```diff
@@ -68,8 +68,12 @@ mod thread {
 }
 ```
 
-Modules implicitly have a [use declaration] specified by crate's [prelude]. The
-[use declaration] can be removed by using the `no_implicit_prelude` [attribute].
+Modules implicitly have some paths in scope. These paths are to built-in types,
```

Comment:
"paths" -> "names", names are in scope.

Reply:
Path is a higher level thing in general, the first segment is searched in the current scope, the following segments are searched in modules (or in impls if they are associated items). Path as a whole being in scope doesn't make much sense.